src/cpu.cpp

```diff
@@ -214,7 +214,7 @@ int cpu_support_arm_asimdhp()
 
 int cpu_support_x86_avx2()
 {
-#if defined(__x86_64__)
+#if defined(__x86_64__) || defined(_WIN32)
```

Comment:
_M_AMD64 Defined as the integer literal value 100 for compilations that target x64 processors. Otherwise, undefined.
https://docs.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=vs-2019
https://sourceforge.net/p/predef/wiki/Architectures/